README.md

```diff
@@ -203,6 +204,16 @@ This command would generate the following configuration:
 }
 ```
 
+### Debugging rails applications
```

Comment:
```suggestion
### Debugging live processes
```

Reply:
For people searching for Rails debugging I suggest keeping at least `Debugging live processes (e.g. Rails applications)`

Reply:
Agreed. Can we switch the title and instead add a phrase explaining what this is then? Something along the lines of

`Instead of launching a process to debug every time, you may want to attach the VS Code debugger to an existing process, such as a Rails server. Follow these instructions to do so.`